shinydashboard/lantern/modules/capabilitymodule.R

```diff
@@ -0,0 +1,60 @@
+# Capability Module
+library(treemapify)
+
+capabilitymodule_UI <- function(id) {
+  
+  ns <- NS(id)
+  
+  tagList(
+    h1("FHIR Resource Types"),
+    p("This is the list of FHIR resource types reported by the capability statements from the endpoints."),
+    fluidRow(
+      column(width=5,
+             tableOutput(ns("resource_type_table"))),
+      column(width=7,
+             h4("Resource Count"),
+             plotOutput(ns("resource_bar_plot"))
+      )
+    )
+  )
+}
+
+capabilitymodule <- function(
+  input, 
+  output, 
+  session,
+  sel_fhir_version,
+  sel_vendor
+){
+
+  ns <- session$ns
+  endpoint_resource_types <- get_fhir_resources_tbl(db_tables)
+ 
+   selected_fhir_endpoints <- reactive({
+    res <- endpoint_resource_types
+    req(sel_fhir_version(), sel_vendor())
+    if (sel_fhir_version() != ui_special_values$ALL_FHIR_VERSIONS) {
+      res <- res %>% filter(fhir_version == sel_fhir_version())
+    }
+    if (sel_vendor() != ui_special_values$ALL_VENDORS) {
+      res <- res %>% filter(vendor_name == sel_vendor())
+    }
+    res
+  })
+  #  endpoint_resource_count <- endpoint_resource_types %>% group_by(type,fhir_version) %>% count() %>% rename(Resource=type,Endpoints=n)
```

Comment:
Remove old code, though it was helpful to me to see that erc was short for `endpoint_resource_count`

Reply:
remove commented code, rename variable